crates/test_utils/wasm/wasm_workspace/create_entry/src/lib.rs

```diff
@@ -69,3 +74,10 @@ fn validate_create_entry_post(
     };
     Ok(r)
 }
+
+#[hdk_extern]
+fn my_activity(_: ()) -> ExternResult<AgentActivity> {
+    let agent = agent_info!()?.agent_latest_pubkey;
+    let query = QueryFilter::new();
+    Ok(get_agent_activity(agent, query, ActivityRequest::Full)?)
+}
```

Comment:
I wonder about the usefulness of this. It's just plugging in their own agent key, and disallowing any kind of querying or filtering. Maybe if this guaranteed getting only local activity this could be useful, but this might even go out to the network.